tests/unit/Core/Framework/MessageQueue/Middleware/RoutingOverwriteMiddlewareTest.php

```diff
@@ -131,15 +131,15 @@ public static function dispatchProvider(): \Generator
         ];
 
         yield 'Default config, no stamps, message in envelope, envelope with stamp' => [
-            'message' => (new Envelope(new AsyncMessage()))->with(new DelayStamp(5)),
+            'message' => (new Envelope(new AsyncMessage()))->with(new BusNameStamp('test')),
```

Comment:
it is not relevant which stamp is used for testing this. Removed the DelayStamp usage due to the new PHPStan rule